From a918a3cd2a130d1d7eb2d7a2dc43c14f3efae11c Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Sat, 6 Jan 2018 00:59:59 +0100
Subject: [PATCH 3/7] drm_hwcomposer: Update external connectors list

DVID, DVII and VGA are required by discrete and integrated GPUs.

Signed-off-by: Mauro Rossi <issor.oruam@gmail.com>

Conflicts:
	drmconnector.cpp

Change-Id: I11235e99c3e91cb009b839da67d9bd567da07637
---
 drmconnector.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drmconnector.cpp b/drmconnector.cpp
index ccb38e2..6c6b1a1 100644
--- a/drmconnector.cpp
+++ b/drmconnector.cpp
@@ -72,6 +72,12 @@ bool DrmConnector::built_in() const {
          type_ == DRM_MODE_CONNECTOR_DSI || type_ == DRM_MODE_CONNECTOR_VIRTUAL;
 }
 
+bool DrmConnector::external() const {
+  return type_ == DRM_MODE_CONNECTOR_HDMIA || type_ == DRM_MODE_CONNECTOR_DisplayPort ||
+         type_ == DRM_MODE_CONNECTOR_DVID || type_ == DRM_MODE_CONNECTOR_DVII ||
+         type_ == DRM_MODE_CONNECTOR_VGA;
+}
+
 int DrmConnector::UpdateModes() {
   int fd = drm_->fd();
 
-- 
2.17.1

